library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -361,6 +405,16 @@ public Slice get(Slice key, long timeBucket, ReadSource readSource)
      */
     private BucketedValue getValueFromTimeBucketReader(Slice key, long timeBucket)
     {
+      if (bloomFilter != null) {
```

Comment:
First get is get from memory, which don't take too much time. And if we take check for the first get, which mean we need to put the flag at the first put(), which probably decrease the accurate very quickly